app/Controller/SongsController.php

```diff
@@ -194,8 +194,8 @@ public function albums() {
                 'Song' => array(
                     'fields' => array('Song.id', 'Song.band', 'Song.album', 'Song.cover'),
                     'conditions' => $subQuery,
-                    'order' => 'Song.album',
-                    'limit' => 36
+                    'order' => ['Song.band', 'Song.album'],
+                    'limit' => 60
```

Comment:
I've looked at the CakePHP thread you gave, I'll integrate the fix and propose a PR to Sonerezh.